app/screens/settings/about/about.tsx

```diff
@@ -182,11 +196,17 @@ const About = ({componentId, config, license}: AboutProps) => {
             const server = buildNumber === version ? intl.formatMessage({id: 'settings.about.server.version.noBuild', defaultMessage: 'Server Version: {version}'}, {version}) : intl.formatMessage({id: 'settings.about.server.version', defaultMessage: 'Server Version: {version} (Build {buildNumber})'}, {version, buildNumber});
             const database = intl.formatMessage({id: 'settings.about.database', defaultMessage: 'Database: {driverName}'}, {driverName: config.SQLDriverName});
             const databaseSchemaVersion = intl.formatMessage({id: 'settings.about.database.schema', defaultMessage: 'Database Schema Version: {version}'}, {version: config.SchemaVersion});
-            const copiedString = `${appVersion}\n${server}\n${database}\n${databaseSchemaVersion}`;
+            let copiedString = `${appVersion}\n${server}\n${database}\n${databaseSchemaVersion}`;
+
+            if (license.IsLicensed === 'true' && loadMetric !== null) {
```

Comment:
Is there a point to re-check for isLicensed here? As without a license loadMetric should always be null